share/spice/timer/timer.js

```diff
@@ -13,7 +13,25 @@ License: CC BY-NC 3.0 http://creativecommons.org/licenses/by-nc/3.0/
     env.ddg_spice_timer = function(api_result) {
         
         //prevent flash of unstyled content if spice shows before css loads
-        $('link[href$="timer.css"]').load(removeHiddenAttr);
+        function waitForCss() {
+            var interval = setInterval(findTimerCss, 10),
+                tries = 0;
+
+            function findTimerCss() {
+                tries++;
+                if ($('link[href$="timer.css"]')[0]) {
```

Comment:
@kethinov This didn't seem to fix the flash, perhaps when this condition is met, you should attach the `removeHiddenAttr` function rather than call it?

![out](https://cloud.githubusercontent.com/assets/4411471/5596763/512b04a2-9264-11e4-9dd8-e493aa763c3c.gif)